CelesteNet.Client/Entities/Ghost.cs

```diff
@@ -216,8 +216,11 @@ public Ghost(CelesteNetClientContext context, DataPlayerInfo playerInfo, PlayerS
                 HoldableAdded = false;
                 Remove(Holdable);
             }
-
-            Alpha = 0.875f * ((CelesteNetClientModule.Settings.PlayerOpacity + 2) / 6f);
+            if(CelesteNetClientModule.Settings.PlayerOpacity == 0){
+                Alpha = 0f;
+            }else{
+                Alpha = 0.875f * ((CelesteNetClientModule.Settings.PlayerOpacity + 2) / 6f);
+            }
```

Comment:
Thanks for the pull request :) It'd be great if you could fix the spacing to match the rest of the code:

```suggestion
            if (CelesteNetClientModule.Settings.PlayerOpacity == 0) {
                Alpha = 0f;
            } else {
                Alpha = 0.875f * ((CelesteNetClientModule.Settings.PlayerOpacity + 2) / 6f);
            }
```

In the worst case I'll just fix the spacing myself before or after hitting the merge button.